lib/assessment.rb

```diff
@@ -13,28 +16,31 @@ def longest(enum)
   # enumerable supplied (i.e. apply the `+` operator)
   # Assume each entry implements a `+` method
   def sum(enum)
-    # Your code goes here
+    (5..10).inject { |sum, n| sum + n }
```

Comment:
Again, you're correct in using  `+` and `inject` (or, equivalently, `reduce`), but we want to operate on enum.
`enum.inject { |sum, n| sum + n  }`